dist/docker/redhat/Dockerfile

```diff
@@ -28,7 +28,8 @@ ADD commandlineparser.py /commandlineparser.py
 ADD docker-entrypoint.py /docker-entrypoint.py
 ADD node_exporter_install /node_exporter_install
 # Install Scylla:
-RUN curl http://downloads.scylladb.com/rpm/centos/scylla-3.1.repo -o /etc/yum.repos.d/scylla.repo && \
+ARG SCYLLA_REPO_URL=http://downloads.scylladb.com/rpm/centos/scylla-3.1.repo
```

Comment:
Let's move this `ARG` definition to the beginning of the `Dockerfile` where it's easier to discover. Let's also add a comment on top of it explaining what it does.